app/src/gui/components/record/relationships/RelatedInformation.tsx

```diff
@@ -321,24 +337,29 @@ export async function get_RelatedFields_for_field(
   multiple: boolean,
   related_type_label: string | undefined,
   form_type: string | undefined,
-  realtion_type: string
+  relation_type: string
```

Comment:
It'd be worth refactoring this function to use a named object input rather than ordered params given how many there are. 

E.g. 

```
function getRelatedRecords({argA, argB} : {argA: typeA, argB: typeB}) {
...
}
```

Reply:
Reduced the number of required params here.